cookbook/docs/Makefile

```diff
@@ -4,7 +4,7 @@ rsts:
 
 .PHONY: html
 html:
-	sphinx-build ${SPHINXOPTS} -W -n -b html . _build
+	sphinx-build ${SPHINXOPTS} -n -W -vvv -b html . _build
```

Comment:
just curious, the order matters here?

Reply:
oh, this just an artifact of debugging... will revert in subsequent PRs